packages/ncids-js/src/components/usa-site-alert/utils/site-alert-close-button.component.ts

```diff
@@ -74,16 +47,13 @@ export class NCICloseButton {
 		this.removeEventListeners();
 
 		// Unset cookie
-		document.cookie = `NCISiteAlert=; Path=${this.options.cookiePath}; Expires=Thu, 01 Jan 1970 00:00:01 GMT`;
+		document.cookie = `NCISiteAlert${this.element.id}=; Path=${this.options.cookiePath}; Expires=Thu, 01 Jan 1970 00:00:01 GMT`;
```

Comment:
Is the requirement a session cookie or a persistent? IIRC, adding a date to a cookie turns it into a persistent cookie. Additionally, on unset, I don't think we should clear out the cookie? (Since it is meant to persist across pages and such...)

Reply:
Actually, yeah, removing the component should not remove the cookie.